x-pack/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/index.test.tsx

```diff
@@ -211,8 +208,8 @@ describe('CustomLink', () => {
         addFilterField(component, 2);
         expect(component.getAllByText('service.name').length).toEqual(4);
       });
-      it('removes items already selected', async () => {
-        const component = await openFlyout();
```

Comment:
Don't we need to `await` this anymore?

Reply:
It's not needed, it was async because I was checking if `callApmApiSpy` was called to continue. But it's causing too much troubles.